include/caffe/common.hpp

```diff
@@ -229,6 +229,9 @@ class MemoryHandlerActivator {
       using_pool_ = true;
       MemoryHandler::usePool();
       MemoryHandler::setGPUs(gpus);
+      void* temp;
+      MemoryHandler::mallocGPU(&temp, 4);
+      MemoryHandler::freeGPU(temp);
```

Comment:
This needs to be wrapped in a `#ifndef CPU_ONLY` (or `#ifdef USE_CNMEM`) for the CPU-only build.